pkg/container/client.go

```diff
@@ -292,6 +295,39 @@ func (client dockerClient) HasNewImage(ctx context.Context, container Container)
 	return true, newImageID, nil
 }
 
+func (client dockerClient) OldEnoughImage(container Container, latestImage t.ImageID) bool {
+	ctx := context.Background()
+
+	updateDelay := client.updateDelay
+	if containerDelay, ok := container.getLabelValue(updateDelayLabel); ok {
+		labelDelay, err := time.ParseDuration(containerDelay)
+
+		// Should we return false or true here?
+		if err != nil {
+			log.Debugf("Update Delay label is invalid for %s", container.Name())
+			return false
```

Comment:
```suggestion
		// Should we return false or true here?
		if err != nil {
			log.WithField("labelValue", containerDelay).Warnf("Update delay label is invalid for %q", container.Name())
			return false
```
If someone explicitly tried to prevent the container from being updated too early, skipping it and including the warning in the notification/log is probably the safest bet (returning false).